### PR TITLE
3.4 bugfixes

### PR DESCRIPTION
**Pull request by kdmnk**
Some minor fixes that came in with 3.4. This can be a hotfix to 3.4.1 if we want.
- view all users on Secretariat/UserController@list
- floating button fix 
- dark mode fix
- delete workshop tag: added policy
Do we have something other than these that should be fixed?

### CODE REVIEW

File: app/Http/Controllers/Secretariat/UserController.php [hunk 1]
```diff
@@ -115,8 +114,7 @@ public function setCollegistType(Request $request)
     public function list()
     {
         $this->authorize('viewAny', User::class);
-        $users = User::role(Role::COLLEGIST)
-            ->with(['roles', 'workshops', 'educationalInformation', 'allSemesters'])->orderBy('name')->get();
+        $users = User::all()->with(['roles', 'workshops', 'educationalInformation', 'allSemesters'])->orderBy('name')->get();
```
Comment (luksan47) [inline app/Http/Controllers/Secretariat/UserController.php:117]: This is not right, if we want to get all users, just leave the `all()` call.
See the difference is that before calling `all`, `get`, `first`, `last`, etc. we have a Builder instance. After, that, we get a collection instance. For query Builders we can say which columns we want to include in the query, but for collections it's too late.

Also, I am not sure if we want to include all users here. Maybe having one for the secretary and having one for the dormitory staff would make sense? I think collegists and tenants should be handled differently.

File: resources/views/user/workshop_tags.blade.php [hunk 2]
```diff
@@ -2,7 +2,11 @@
 @if($user->hasEducationalInformation())
 @foreach ($user->workshops as $workshop)
     <span class="new badge {{ $workshop->color() }} scale-transition" id="user-workshop-{{ $user->id }}-{{ $workshop->id }}" data-badge-caption="">
-        <nobr>{{$workshop->name}} <span style="cursor: pointer;" onclick="deleteWorkshop({{ $user->id }}, {{ $workshop->id }})">&cross;</span></nobr>
+        <nobr>{{$workshop->name}}
```
Comment (luksan47) [inline resources/views/user/workshop_tags.blade.php:5]: BTW, would removing nobr look good/bad? Would it solve the too long name issue with Mendöl?

**Review by luksan47 — CHANGES_REQUESTED** [review]
Left some comments, otherwise LGTM. We can create a minor release.

File: app/Http/Controllers/Secretariat/UserController.php (hunk 1 above)
Comment (kdmnk) [reply app/Http/Controllers/Secretariat/UserController.php:117]: Okay, I will restore the original one.

File: resources/views/user/workshop_tags.blade.php (hunk 2 above)
Comment (kdmnk) [reply resources/views/user/workshop_tags.blade.php:5]: I think multilined tags look bad. However, we can try switching from tables to divs.

Comment (luksan47) [reply resources/views/user/workshop_tags.blade.php:5]: Before switching, let's think about different options.

Comment (luksan47) [reply resources/views/user/workshop_tags.blade.php:5]: Maybe trimming long names would be enough.

**Review by luksan47 — APPROVED** [review]
LGTM!
Let's do a minor release, I'll collect everything important.

**Pull request merged**